app/src/main/java/com/example/perfectweatherallyear/ui/detailWeather/DetailWeatherFragment.kt

```diff
@@ -14,6 +15,9 @@ const val ARG_DAY_WEATHER: String = "DAY_WEATHER"
 class DetailWeatherFragment : Fragment() {
     private var _binding: FragmentDetailWeatherBinding? = null
     private val binding get() = _binding!!
+    private val detailWeatherViewModel: DetailWeatherViewModel by lazy {
+        ViewModelProvider(this).get(DetailWeatherViewModel::class.java)
+    }
```

Comment:
Can be simplified with help of [fragment-ktx](https://developer.android.com/kotlin/ktx#fragment)